tests/test_vector_linz/test_vector_linz.py

```diff
@@ -20,8 +21,8 @@ class LinzVectorsTest(unittest.TestCase):
     OpenTopography within a small region. All files are deleted after checking their names and size.
 
     Tests run include:
-        1. test_railways - Test that the expected railways dataset is downloaded from LINZ
-        2. test_pastural_lease - Test that the expected pastural lease dataset is downloaded from LINZ
+        * test_50781 - Test layer 50781 features are correctly downloaded
+        * test_51572 - Test layer 51572 features are correctly downloaded
     """
```

Comment:
I think this a good way of documenting the tests but it is an extra step of repeating the same documentation that goes in the docstrings, Is it intended to be a subset? The numbers seem less clear than the original name though

Reply:
I could change it to:
test_#### - Test the layer features are correctly downloaded.

I ended up going with the layer number as that's the key for determining which vector I'm actually looking at. My text descriptions were a bit ad-hoc where the layer id is unique. Thoughts?

Reply:
Ok - I have given a go at updating the test information to hopefully make it easier to link the layer number to the layer description and to reduce fiddly repeated text.